gemato/openpgp.py

```diff
@@ -15,6 +15,9 @@
 
 import gemato.exceptions
 
```

Comment:
I think two blank lines here are the standard but I could be wrong.

Reply:
Yes indeed. I should have PEP8 it.